packages/webpack/index.js

```diff
@@ -12,6 +12,12 @@ const configure = (config, options) => ({
   getBuildConfig(...args) {
     return bootstrap(config, options).getBuildConfig(...args);
   },
+  getDevelopConfig(...args) {
```

Comment:
Maybe we should consider renaming these to `getWebpack*Config` instead, to make it more clear, what these hooks do.